### PR TITLE
[capture] Removes variable reset_firmware

### LINKED ISSUE #105: [ecdsa] Don't reset target between traces
It seems the target is currently reset between capturing different traces or trace segments which leads to very low performance as the binary needs to be transferred over SPI for every segment. We should look into changing the capture setup to not require a reset of the target between segments.

### PR DESCRIPTION
**Pull request by vrozic**
Code-cleanup for capture.py.
Removes variable reset_firmware as discussed in issue #105 

Closes #105

### CODE REVIEW

**Review by vogelpi — APPROVED** [review]
LGTM, thanks @vrozic !

File: cw/capture.py [hunk 1]
```diff
@@ -1772,17 +1762,10 @@ def capture_ecdsa_sections(ot, fw_bin, pll_frequency, num_sections, secret_k, pr
...
     """
 
-    # This determines if we want to reprogram the firmware
-    # before every sign operation.The default value is false.
-    reset_firmware = False
-
     # Create a temporary buffer to keep the collected sections
     tmp_buffer = np.array([])
     for ii in range(num_sections):
         # Reflash the firmware and set the CW310 pll frequency
```
Comment (vogelpi) [inline cw/capture.py:1768]: I think also this comment needs to be removed. Similar occurrence below.

File: cw/capture.py [hunk 2]
```diff
@@ -2133,9 +2112,6 @@ def capture_ecdsa_stream(ot, fw_bin, pll_frequency, capture_cfg):
         # Create an arrey to keep the traces
         waves = np.array([])
         # Reflash the firmware and set the CW310 pll frequency
```
Comment (vogelpi) [inline cw/capture.py:2114]: See here

File: cw/capture.py (hunk 1 above)
Comment (vrozic) [reply cw/capture.py]: Agreed, I've removed this comment in both places with the latest change

**Pull request merged**